tools/oeedger8r/CMakeLists.txt

```diff
@@ -26,7 +26,8 @@ add_custom_command(
   COMMAND cmake -E copy_directory ${CMAKE_CURRENT_SOURCE_DIR}/intel ${CMAKE_CURRENT_BINARY_DIR}/intel
   COMMAND cmake -E copy_directory ${CMAKE_CURRENT_SOURCE_DIR}/src ${CMAKE_CURRENT_BINARY_DIR}/src
   COMMAND cmake -E copy_if_different ${CMAKE_CURRENT_SOURCE_DIR}/dune-project ${CMAKE_CURRENT_SOURCE_DIR}/package.json ${CMAKE_CURRENT_BINARY_DIR}
-  COMMAND esy
+  COMMAND esy install
```

Comment:
Why do we need to `esy install`?

Reply:
`esy` is equivalent to `esy install && esy build`. Running `esy install` is a necessary step to build; it resolves the `package.json` dependencies (such as OCaml etc.) and makes them available in the build environment. This is more again to `pyenv`, not `apt`: it's not a system-wide "install" it's an "install my dependencies for this environment" step (modeled after `npm` really).